### PR TITLE
WIP #86, added the foundation

### PR DESCRIPTION
**Pull request by MadsHeilmann**
Added the sections and populated introduction.
Added textidote command for the process analysis in the READ.ME

### CODE REVIEW

**Review by TexWO — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -17,10 +17,18 @@ chktex -q -l .chktexrc src/main.tex
...
+Report:
+
 ```shell
 textidote src/main.tex
 ```
 
+Process analysis:
+
+```shell
+textidote src/process-analysis/process-analysis.tex
```
Comment (audio-engineer) [inline README.md:29]: Why is this necessary?

File: src/process-analysis/counsellor.tex [hunk 2]
```diff
@@ -0,0 +1,3 @@
+% textidote: ignore begin
+\section{Working with the counselor}\label{sec:working-with-the-counselor}
```
Comment (audio-engineer) [inline src/process-analysis/counsellor.tex]: Supervisor

File: src/process-analysis/process-analysis.bib [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@report{P1,
+    author = {Kristiyan Mariyan Georgiev and Mads Heilmann and Martin Kedmenec and Sebastian Hemmer Bech Mygind and
+ Hasnain Ali Sajad and Jawad Mehmood Khan Qayyum and Simon Woidemann},
```
Comment (audio-engineer) [inline src/process-analysis/process-analysis.bib:3]: !

**Review by audio-engineer — CHANGES_REQUESTED** [review]

File: src/process-analysis/process-analysis.bib (hunk 3 above)
Comment (MadsHeilmann) [reply src/process-analysis/process-analysis.bib:3]: Is it the indentation that is wrong here?

File: README.md (hunk 1 above)
Comment (MadsHeilmann) [reply README.md:29]: Using the command "textidote src/main.tex" does not analyze the documents related to the process analysis, why i found it nice to have the other command at hand in the READ.ME

File: src/process-analysis/process-analysis.bib (hunk 3 above)
Comment (audio-engineer) [reply src/process-analysis/process-analysis.bib:3]: the names are wrong

**Review by audio-engineer — CHANGES_REQUESTED** [review]

File: src/process-analysis/process-analysis.bib (hunk 3 above)
Comment (MadsHeilmann) [reply src/process-analysis/process-analysis.bib:3]: What names are wrong specifically?

Comment (audio-engineer) [reply src/process-analysis/process-analysis.bib:3]:
```suggestion
and Simon Woidemann},
```

**Review by audio-engineer — CHANGES_REQUESTED** [review]

File: src/process-analysis/process-analysis.bib (hunk 3 above)
Comment (MadsHeilmann) [reply src/process-analysis/process-analysis.bib:3]: They were authors on the P1 project though, why they should be included in the list of authors

**Review by SebastianMygind — APPROVED** [review]

**Review by audio-engineer — APPROVED** [review]
